ctapipe/reco/hillas_reconstructor.py

```diff
@@ -410,35 +342,23 @@ def estimate_direction(self, hillas_planes):
         crossings : shape (n,3) list
             an error estimate
         """
+        index_a, index_b = np.array(list(combinations(range(len(norm)), 2))).T
+        crossings = np.cross(norm[index_a], norm[index_b])
+        mask = crossings[:, 2] < 0
+        crossings[mask] = -crossings[mask]
```

Comment:
Why not directly do `np.cross(norm[index_b], norm[index_a])`?

Reply:
what would that solve?

Reply:
switching the argument order inverts the sign for all vectors, which still leaves you with *some* vectors pointing down.

Reply:
Ah right, I misunderstood the purpose of this